### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Oscah01**
- Add missing jsx elements (without adding extra functionality)
- add styling

### CODE REVIEW

File: src/pages/Books.jsx [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+  }, [dispatch]);
+
+  const booklists = useSelector((state) => state.books.books);
+  return (
+    <div>
+      <div>
+        {booklists.map((item) => (
+          <Book key={item.item_id} item={item} />
+        ))}
+      </div>
```
Comment (DuaneDave) [inline src/pages/Books.jsx:20]: The styling of the app is still not matching the Zeplin design except i zoom out to 80%, I have circled out some parts of your design that is not consistent with Zeplin. Please work on the design to make your app more visually appealing.


YOUR DESIGN

bsdcomplete

ZEPLIN DESIGN

incomplete

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Oscah01  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Correct usage of workflows
✔️ No linter error
✔️ PR summary is descriptive

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -1,27 +1,263 @@
...
+  gap: 2rem;
+  padding: 2.9rem 10rem;
+  background-color: #fafafa;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  margin-bottom: 1.9rem;
+}
+
+h1 {
+  color: #0290ff;
+}
```
Comment (levy002) [inline src/index.css:44]: - [ ] As requeted by the previous reviewer, Kindly make sure to stick to the provided Figma design. Please make sure the `BOOK STORE` title is aligned on the same starting vertical line with the booklist and form on the page as previously shown on the designs.

   - ### YOUR DESIGN
   
    image

   - ### FIGMA DESIGN
   
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Oscah01   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**